<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ĐĂNG KÝ DỊCH VỤ</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>311</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Thông tin khách hàng</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Họ và tên</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ho_va_ten">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Giới tính</string>
    </property>
   </widget>
   <widget class="QComboBox" name="gioi_tinh">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Nam</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nữ</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Thông tin dịch vụ</string>
   </property>
   <widget class="QCheckBox" name="sms">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SMS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="video_call">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Video call</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="four_g">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4G</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_goi_nho">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Báo gọi nhỡ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Số phút gọi</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="so_phut_goi">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dung lượng net</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="dung_luong_net">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="dang_ky">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Đăng ký</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gia_thue_bao">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>vnd</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
